app/jobs/cron_job/synchronize_crm.rb

```diff
@@ -20,28 +20,36 @@ def perform
 
     client.database_query(database_id: NOTION_DATABASE_ID, filter:) do |page|
       page.results.each do |notion_page|
-        rdv_count = Rdv.where(organisation: organisations_ids(notion_page.properties["COMPTE PROD"].url)).count
-        client.update_page(page_id: notion_page.id, properties: { "NOMBRE DE RDV" => rdv_count })
+        ids = organisations_ids(notion_page.properties["COMPTE PROD"].url)
+        if ids.blank?
+          client.update_page(page_id: notion_page.id, properties: { "NOMBRE DE RDV" => nil })
+        else
+          rdv_count = Rdv.where(organisation: ids).count
+          client.update_page(page_id: notion_page.id, properties: { "NOMBRE DE RDV" => rdv_count })
+        end
```

Comment:
petite suggestion un peu plus DRY : 

```suggestion
        rdv_count = ids.blank? ? 0 : Rdv.where(organisation: ids).count
        client.update_page(page_id: notion_page.id, properties: { "NOMBRE DE RDV" => rdv_count })
```